blog.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+<head>
+  <meta charset="utf-8">
+  <title>>Adopt A Dog</title>
+
+  <link rel="stylesheet" type="text/css" href="./styles/blog.css">
+
+</head>
+
+<body>
+  <header>
+    <img src="./images/adoptadog-logo.svg" alt="adopt a dog logo">
+    <h1>Seattle's Premiere dog adoption agency</h1>
+    <img src="./images/cart-icon.svg" alt="shopping cart icon">
+    <p>check out</p>
+  </header>
+
+  <main>
+    <nav class="col span_3_of_12">
+      <ul class="menunavigation">
+        <li class="menunavigationitem"><a href="./index.html">Home</a></li>
+        <li class="menunavigationitem"><a href="./doglist.html">Dogs</a></li>
+        <li class="menunavigationitem"><a href="./blog.html">Blog</a></li>
+      </ul>
+    </nav>
+    <h2>Adptadog Blog</h2>
+    <section>
+      <img src="./images/blog-1.jpg" alt="person watching view with dog">
```

Comment:
You'll want to wrap these blog articles with some sort of wrapping element. Can you think of an HTML5 element for this kind of thing?